### ISSUE TITLE
Can't select category for adding or editing product

### ISSUE THREAD

**Issue by Manu270891** [amejiarosario/meanshop#49]
When creating or editing a product, there is no category field for selecting a category. How can we implement that?

**Comment by amejiarosario** [comment]
You can list the Categories in a dropdown and associate them with the product. Take a look at `seed.js` to see how to associate products with categories

**Comment by Manu270891** [comment]
Can you give me an example code please?

**Comment by merry75** [comment]
yes an example of code to set up categories in the product controller would be appreciated, having a hard time on that

**Comment by costadanilofreitas** [comment]
Any answers?
